test/build-test.ts

```diff
@@ -26,7 +26,7 @@ describe("build", () => {
     if (isEmpty(path)) continue;
     const only = name.startsWith("only.");
     const skip = name.startsWith("skip.");
-    const outname = only || skip ? name.slice(5) : name;
+    const outname = name.replace(/^only\.|skip\./, "");
```

Comment:
This lets you change the definition of `only` without a surprising rename of the test. 🙂